prow/plugins/golint/suggestion/golint_suggestion.go

```diff
@@ -26,12 +26,14 @@ var (
 	lintNamesUnderscoreRegex = regexp.MustCompile("don't use underscores in Go names; (.*) should be (.*)")
 	lintNamesAllCapsRegex    = regexp.MustCompile("don't use ALL_CAPS in Go names; use CamelCase")
 	lintStutter              = regexp.MustCompile("name will be used as [^.]+\\.(.*) by other packages, and that stutters; consider calling this (.*)")
+	lintErrorfRegex          = regexp.MustCompile("should replace (errors.New\\(fmt.Sprintf\\(\\.\\.\\.\\)\\)) with (fmt.Errorf\\(\\.\\.\\.\\))")
```

Comment:
Consider using backtick characters (``` ` ```) instead of quotes (``` " ```) to define regexp strings. Using backticks makes the string a raw literal, removing the need to double escape.
More details about string literals: https://golang.org/ref/spec#String_literals

Also note that the `.` in `errors.New` needs to be escaped from the regexp parsing so that it is treated as a literal period.
```suggestion
	lintErrorfRegex          = regexp.MustCompile(`should replace (errors\.New\(fmt.Sprintf\(\.\.\.\)\)) with (fmt\.Errorf\(\.\.\.\))`)
```